sdk/core/Azure.Core.TestFramework/src/TestRecordingMismatchException.cs

```diff
@@ -13,16 +13,25 @@ public TestRecordingMismatchException()
         {
         }
 
-        public TestRecordingMismatchException(string message) : base(message)
+        public TestRecordingMismatchException(string message) : base(AppendReminder(message))
         {
         }
 
-        public TestRecordingMismatchException(string message, Exception innerException) : base(message, innerException)
+        public TestRecordingMismatchException(string message, Exception innerException) : base(AppendReminder(message), innerException)
         {
         }
 
         protected TestRecordingMismatchException(SerializationInfo info, StreamingContext context) : base(info, context)
         {
         }
+
+        private static string AppendReminder(string message)
+        {
+            const string Reminder = "Have you forgotten to push recordings to the assets repository?";
```

Comment:
It may be a good idea to add a link to the Test Framework docs once they're merged.